src/pidfile.c

```diff
@@ -38,14 +38,13 @@
  * @copyright 2-clause BSD License
  */
 
-#include <sys/stat.h>		/* utimensat() */
-#include <sys/time.h>		/* utimensat() on *BSD */
 #include <sys/types.h>
 #include <errno.h>
 #include <paths.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#include <utime.h>
```

Comment:
I need to check this with all the supported targets, but otherwise tentatively approve.